CMakeLists.txt

```diff
@@ -0,0 +1,217 @@
+cmake_minimum_required(VERSION 3.7.0)
+
+project(dcos-mesos-modules)
+
+set(MESOS_ROOT "../mesos" CACHE PATH "Mesos root directory.")
+get_filename_component(MESOS_ROOT ${MESOS_ROOT} ABSOLUTE)
+set(BOOST_ROOT_DIR "" CACHE PATH "Boost libs root directory.")
+
+message("MESOS_ROOT is ${MESOS_ROOT}")
+
+# CMAKE MODULE SETUP.
+#####################
+# Paths that are searched when `include(...)` is called.
```

Comment:
I'm a bit lost: among the three `include(...)`s I see in this patch
```
include(CTest)
include(Versions OPTIONAL)
include(Dependencies)
``` 
which one is included from ${MESOS_ROOT}/...?

Reply:
`Versions`; it will not be necessary if we compile with an installed version of mesos. on hold for now until decision whenever require mesos install for module development.

Reply:
Now that we settled on using -DENABLE_INSTALL_MODULE_DEPENDENCIES, can `Versions` be just dropped?

Reply:
yes, removed.